README.md

```diff
@@ -13,8 +13,12 @@ Included Characterization Tools
 
 The robot characterization toolsuite currently supports chracterization for:
 
-- Drivetrains
-- Arms
+* [Java](robot-java)
+* [Java TalonSRX](robot-java-talonsrx)
+* [Python](robot-python)
+* [Python TalonSRX](robot-python-talonsrx)
+* [C++](robot-cpp)
+* [C++ TalonSRX](robot-cpp-talonsrx)
```

Comment:
I think you want to change this only under the subdirectory readmes.

Reply:
Whoops, my bad. Fixed this in latest commit.